bibliopixel/remote/control.py

```diff
@@ -0,0 +1,140 @@
+import multiprocessing
+import inspect
+from .. animation import BaseAnimation
+from .. animation.animation import COMPLETE_REASON
+from . import server
+from .. import log
+
+
+DEFAULT_OFF = 'OFF_ANIM'
+DEFAULT_ANIM_CONFIG = {
+    'bgcolor': '#00ff00',
+    'font_color': '#ffffff',
+    'display': None
+}
+
+
+class RemoteControl():
+    def __init__(self, base_config, animations):
```

Comment:
Of course it's not a sequence!!! base_config is everything except the animations. All the server stuff:
```
        "external_access": true,
        "port": 5000,
        "title": "Test Remote",
        "bgcolor": "linear-gradient(to bottom, #f36926, black)",
```
Has nothing to do with animations! I can change to config though. Don't really care. `animations` is the list of pre-built animation objects that already have their Runner and stuff set.